Documentation/policy/language.rst

```diff
@@ -527,18 +527,29 @@ if all fields are empty or missing, the rule will match all Kafka messages.
 
 The following fields can be matched on:
 
-
-APIVersion
-  APIVersion is the version matched against the api version of the Kafka
-  message. If set, it has to be a string representing a positive integer. If
-  omitted or empty, all versions are allowed.
+Role
+  Role is a case-insensitive string and describes a group of API keys
+  necessary to perform certain higher level Kafka operations such as "produce"
+  or "consume". A Role automatically expands into all APIKeys required
+  to perform the specified higher level operation.
+
+  The following values are supported:
+	- "produce": Allow producing to the topics specified in the rule
+	- "consume": Allow consuming from the topics specified in the rule
+  This field is incompatible with the APIKey field, i.e either APIKey or Role
```

Comment:
`either APIKey or Role can be specified` -> `APIKey and Role cannot both be specified in the same rule`

Reply:
done